include/UIKit/UIImage.h

```diff
@@ -36,95 +40,130 @@
 #import <QuartzCore/CALayer.h>
 
 enum {
-    UIImageOrientationUp,
-    UIImageOrientationDown, // 180 deg rotation
-    UIImageOrientationLeft, // 90 deg CCW
-    UIImageOrientationRight, // 90 deg CW
-    UIImageOrientationUpMirrored, // as above but image mirrored along
-    // other axis. horizontal flip
-    UIImageOrientationDownMirrored, // horizontal flip
-    UIImageOrientationLeftMirrored, // vertical flip
-    UIImageOrientationRightMirrored, // vertical flip
+  UIImageOrientationUp,
+  UIImageOrientationDown,       // 180 deg rotation
+  UIImageOrientationLeft,       // 90 deg CCW
```

Comment:
not context of this CR.

---
In reply to: [84147591](https://github.com/Microsoft/WinObjC/pull/1185#discussion_r84147591) [](ancestors = 84147591)